tiamat/src/androidMain/kotlin/com/composegears/tiamat/Platform.android.kt

```diff
@@ -32,4 +32,11 @@ internal actual fun <Args> NavDestinationScope<Args>.PlatformContentWrapper(
 @Composable
 actual fun NavBackHandler(enabled: Boolean, onBackEvent: () -> Unit) {
     BackHandler(enabled, onBackEvent)
-}
\ No newline at end of file
+}
+
+/**
+ * We can not call T::class in @Composable functions,
+ *
+ * workaround is to call it outside of @Composable via regular inline fun
+ */
+actual inline fun <reified T : Any> className(): String = T::class.qualifiedName!!
```

Comment:
can be internal?

Reply:
Public `rememberViewModel` uses `className` as inline fun, so it should be public